go-controller/pkg/util/net_linux.go

```diff
@@ -240,10 +259,21 @@ func LinkAddrExist(link netlink.Link, address *net.IPNet) (bool, error) {
 
 // LinkAddrAdd adds a new address. If both preferredLifetime & validLifetime,
 // are zero, then they are not applied, but if either parameters are not zero, both are applied.
+// This function adds a label '<bridge_name>-masq' while adding either V4HostMasqueradeIP
+// or V6HostMasqueradeIP to a bridge. This is to help identify old masquerade subnet in case it
+// gets changed at day 2.
 func LinkAddrAdd(link netlink.Link, address *net.IPNet, flags, preferredLifetime, validLifetime int) error {
```

Comment:
Can we just use `netLinkOps.AddrAdd` directly when adding the masq addr instead of hiding it under `LinkAddrAdd`?
Also, it might be useful to extract the label generation into a util function.